pkgs/servers/matrix-synapse/matrix-appservice-irc/default.nix

```diff
@@ -1,16 +1,23 @@
-{ pkgs, nodePackages, makeWrapper, nixosTests, nodejs, stdenv, lib, ... }:
+{ pkgs, nodePackages, makeWrapper, nixosTests, nodejs, stdenv, lib, fetchFromGitHub, ... }:
```

Comment:
We should drop the `...` here. It isn't required and not good practice. I know this isn't your fault but if we change it we can make a slight improvement.